internal/iptables/iptables.go

```diff
@@ -221,7 +221,7 @@ type runner struct {
 func newInternal(exec utilexec.Interface, protocol Protocol, lockfilePath14x, lockfilePath16x string) Interface {
 	version, err := getIPTablesVersion(exec, protocol)
 	if err != nil {
-		klog.InfoS("Error checking iptables version, assuming version at least", "version", MinCheckVersion, "err", err)
+		logrus.Debug("Error checking iptables version, assuming version at least", "version", MinCheckVersion, "err", err)
```

Comment:
Why not a warning here? Or keep it at the same log level. This seems potentially useful for the users.

Reply:
This is the current way to handle klog.InfoS .
Please see https://github.com/cri-o/cri-o/blob/f552e82b023c8e8034a1ecd85bbbecbaa7d61afc/internal/log/klog.go#L12-L38

Reply:
I just wanted to keep the current way, but it makes sense. thanks!